pandas/tests/extension/base/ops.py

```diff
@@ -19,7 +19,7 @@ class BaseOpsUtil:
     divmod_exc: type[Exception] | None = TypeError
 
     def _get_expected_exception(
-        self, op_name: str, obj, other
+        self, op_name: str, obj, other, request
```

Comment:
The `request` in this file don't look like they are used?

Reply:
Fixed by using LSP parameters elsewhere.